simulations/vip-492/basesepolia.ts

```diff
@@ -0,0 +1,86 @@
+import { impersonateAccount, setBalance } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { forking, testForkedNetworkVipCommands } from "src/vip-framework";
+
+import vip491, { RESILIENT_ORACLE_BASE_SEPOLIA } from "../../vips/vip-492/bsctestnet";
+import RESILIENT_ORACLE_ABI from "./abi/ResilientOracle.json";
+
+const { basesepolia } = NETWORK_ADDRESSES;
+
+forking(25337385, async () => {
+  const provider = ethers.provider;
+
+  await impersonateAccount(basesepolia.NORMAL_TIMELOCK);
+  await setBalance(basesepolia.NORMAL_TIMELOCK, ethers.utils.parseEther("1000000"));
+
+  const resilientOracle = new ethers.Contract(RESILIENT_ORACLE_BASE_SEPOLIA, RESILIENT_ORACLE_ABI, provider);
+
+  describe("Pre-VIP behaviour", async () => {
+    it("check USDC price", async () => {
+      expect(await resilientOracle.getPrice("0xFa264c13d657180e65245a9C3ac8d08b9F5Fc54D")).to.equal(
+        parseUnits("0.9999794", 30),
```

Comment:
If we move these values to variables, it would be easier to check that the prices are the same. We could also store the token symbols and addresses into a list, an iterate on it:

```
const prices = [
  {
    symbol: "USDC",
    address: "0xFa2...",
    expectedPrice: parseUnits("0.9999794", 30),
  },...
]
```